plugins/database/src/db_plugin.cpp

```diff
@@ -14499,7 +14507,20 @@ irods::error db_mod_ticket_op(
                     log_sql::debug("chlModTicket SQL 11");
                 }
 
-                return execute_sql("insert into R_TICKET_ALLOWED_HOSTS (ticket_id, host) values (? , ?)");
+                irods::error ret = execute_sql("insert into R_TICKET_ALLOWED_HOSTS (ticket_id, host) values (? , ?)");
+                if (!ret.status()) {
+                    log_sql::error("TODO"); // TODO
```

Comment:
You can get the iRODS error code by calling `ret.code()`.

Please include it in the log message.

_This applies to other locations too._